Source/WebCore/rendering/updating/RenderTreeUpdater.h

```diff
@@ -84,6 +85,7 @@ class RenderTreeUpdater {
     RenderTreePosition& renderTreePosition();
 
     GeneratedContent& generatedContent() { return *m_generatedContent; }
+    ViewTransition& viewTransition() { return *m_viewTransition; }
```

Comment:
Should we check/assert about null pointer here ? (Should we use a pointer in the first place?)

Reply:
I just followed the same pattern as GeneratedContent